docs/use_cases/knowledge_graph_embedding.md

```diff
@@ -0,0 +1,228 @@
+<!-- SEO: Introduction to Knowledge Graphs. Introduction to Knowledge Graph Embedding. Introduction to the DistMult algorithm. Example code for training KGE model. Example code for evaluating KGE model. Example code for answering questions with a KGE model. KGE and LLM comparison on QA task.  -->
+
+# Learning Scemantic Representations for Knowledge Graphs
+
+<!-- TODO: Cover image: 
+1. You can create your own cover image and put it in the correct asset directory,
+2. or you can give an explanation on how it should be and we will help you create one. Please tag arunesh@superlinked.com or @AruneshSingh (GitHub) in this case. -->
+
+In today's world, LLMs (Large Language Models) are everywhere, doing all sorts of language-related tasks really well. They're the go-to solution for understanding and using text in many different ways.
+
+However, in some specific areas, there are other more specialized approaches that might give better results for specific tasks. 
+
+This post digs into when these popular language models might not be the best choice. We'll see how specialized methods, like KGE (Knowledge Graph Embedding) algorithms, could actually be better for certain tasks.
```

Comment:
I'd hint here, that LLM's are great for text, but there are other type of data representations where, maybe not so much

Reply:
I see your approval, but you didn't change the text here, right? Is it in progress, or do you think it should stay like this?

Reply:
I included that in the previous paragraph. Do you want me to emphasize this by including it here too?

Reply:
yes, sorry indeed, I should have looked at the diff not just the paragraph I commented on. Good!